src/mcmc/hmc.py

```diff
@@ -843,9 +843,9 @@ def generate_white_noise(self, size, truncate=True):
         self.random_state = np.random.get_state()
         np.random.set_state(np_state)
 
-        if truncate:
-            randn_vec[randn_vec > 3] = 3
-            randn_vec[randn_vec < -3] = -3
+        # if truncate:
```

Comment:
I don't think we have to comment it out (?) can't we just pass `False` for the argument `truncate` when we call the function?

Reply:
Yes, we can change to let truncate=False